common/models/user.js

```diff
@@ -7,6 +7,41 @@ var config = require('../../server/config.json');
 var path = require('path');
 
 module.exports = function(User) {
+
+//send email again incase of unverified email
+User.triggerEmail = function(userId, cb) {
```

Comment:
I am sorry I missed this part. I have removed the method which I had created to trigger an email and reused POST /api/users/:id/verify . PTAL